src/components/SkipLink/SkipLink.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import cx from 'classnames';
+
+type SkipLinkProps = {
+  className?: string;
+  href: string;
+  isNarrow?: boolean;
+  skipLinkText: string;
+};
+
+export const SkipLink = ({
+  className,
+  href,
+  isNarrow,
+  skipLinkText
```

Comment:
As we are already inside the SkipLink component, would it be more concise/appropriate to name this prop `text`? It might read better when writing:

```
<SkipLink {...props} text="Skip to content" />
```